Sources/PackageCollectionGenerator/PackageCollectionGenerate.swift

```diff
@@ -45,6 +45,9 @@ public struct PackageCollectionGenerate: ParsableCommand {
     @Option(help: "The revision number of the generated package collection")
     private var revision: Int?
 
+    @Flag(name: .long, inversion: .prefixedNo, help: "Format output using friendly indentation and line-breaks.")
+    private var prettyPrinted: Bool = true
```

Comment:
I think the default should be false since pretty-printing adds redundant new lines and whitespaces to the generated JSON document, which means larger download size. Package collections are primarily for machines, and humans can always use tools such as [jq](https://stedolan.github.io/jq/) to make them more readable.

Reply:
Works for me — changed the default back to non-pretty in b0121d9.